secret-engine/src/main/java/com/sequenceiq/cloudbreak/service/secret/service/SecretAspectService.java

```diff
@@ -52,7 +52,7 @@ public Object proceedSave(ProceedingJoinPoint proceedingJoinPoint) {
                     }
                 }
             } catch (IllegalArgumentException e) {
-                LOGGER.error("Given entity isn't instance of TenantAwareResource. Secret is not deleted!", e);
+                LOGGER.error("Given entity isn't instance of AccountIdAwareResource. Secret is not deleted!", e);
```

Comment:
Nit: Perhaps replace the hardwired class name with `AccountIdAwareResource.class.getSimpleName()` like a few lines below in `findAccountId()`.